sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -1279,4 +1279,62 @@ class CSVSuite extends QueryTest with SharedSQLContext with SQLTestUtils {
       Row("0,2013-111-11 12:13:14") :: Row(null) :: Nil
     )
   }
+
+  def checkHeader(multiLine: String): Unit = {
```

Comment:
Could you also test it when setting `header` to `true` and `false`?

Reply:
added